azure-pipelines.yml

```diff
@@ -799,7 +799,19 @@ jobs:
           # the PR. Hence the `|| echo ""`.
           PR_HANDLER=$(head -1 release/rotation | awk '{print $1}')
 
-          tell_slack "<@${PR_HANDLER}> <https://dev.azure.com/digitalasset/daml/_build/results?buildId=$(Build.BuildId)|Build $(Build.BuildId)> for release PR <https://github.com/digital-asset/daml/pull/${PR}|#${PR}> has completed with status $(status). See <https://github.com/digital-asset/daml/blob/master/release/RELEASE.md|RELEASE.md> for what to do next."
+          case "$(status)" in
+            Succeeded*)
+              msg="has succeeded! Next step is to approve & merge."
+              ;;
+            Failed*)
+              msg="has failed. Please investigate."
+              ;;
+            *)
```

Comment:
https://hackage.haskell.org/package/acme-php-0.0.3/docs/Prelude-PHP.html#t:Bool

Reply:
I believe in this case "Skipped" and "Canceled" may also be valid values, but unfortunately the documentation doesn't make it quite clear to me.